docs/guides/ingesters-scaling-up-and-down.md

```diff
@@ -42,6 +42,9 @@ The ingesters scale down is deemed an infrequent operation and no automation is
   - `-blocks-storage.bucket-store.metadata-cache.bucket-index-content-ttl=1m`
   - `-blocks-storage.bucket-store.metadata-cache.tenant-blocks-list-ttl=1m`
   - `-blocks-storage.bucket-store.metadata-cache.metafile-doesnt-exist-ttl=1m`
+- Allow ingester to leave the ring after shutdown
+  - `ingester.unregister-on-shutdown=true`
```

Comment:
Note that this option is only used when **not** calling `/shutdown`, but performing standard shutdown by sending SIGTERM signal. When `/shutdown` endpoint is called, ingester will **always** unregister from the ring.

Reply:
ah, ok this is great, this PR is not needed then, thanks for the info.